### ISSUE TITLE
Versão desatualizada

### ISSUE THREAD

**Issue by homemp7** [fabiovianna89/SINESP_Consulta#6]
Ao tentar realizar a consulta retorna o código 7 com a mensagem de versão desatualizada.

**Comment by marionantes** [comment]
Já existe alguma correção para o problema acima?

**Comment by homemp7** [comment]
Não, pois ninguém solicitou um pull request ainda. Acho até que o projeto foi abandonado =/ Sugiro usar outra linguagem se possível.

**Comment by marionantes** [comment]
@fabiovianna89 

Tem interesse em realizar as novas alterações? Podemos fazer uma contribuição em R$

**Comment by leotx** [comment]
@gusmurf @marionantes 
Caso tenham interesse, disponibilizei uma versão atualizada aqui: https://github.com/leotx/sinesp
